service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -537,5 +504,214 @@
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"BANDWIDTH",
+    "text":"There are so many things wrong about you, that I'm not sure I have enough bandwidth to get through the summary.",
+    "ssml":"There are so many things wrong about you, that I'm not sure I have enough bandwidth to get through the summary.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"There are so many things wrong about [TARGET], that I'm not sure I have enough bandwidth to get through the summary.",
+    "targetableSsml":"There are so many things wrong about [TARGET], that I'm not sure I have enough bandwidth to get through the summary."
+  },
+  {
+    "quipGroup":"SHALLOW_PERSONALITY",
+    "text":"Your personality is as shallow as a dried up puddle.",
+    "ssml":"Your personality is as shallow as a dried up puddle.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET]'s personality is as shallow as a dried up puddle.",
+    "targetableSsml":"[TARGET]'s personality is as shallow as a dried up puddle."
+  },
+  {
+    "quipGroup":"MAGICIAN",
+    "text":"I wish I was a magician, so I could make you disappear.",
+    "ssml":"I wish I was a magician, so I could make you disappear.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"I wish I was a magician, so I could make [TARGET] disappear.",
+    "targetableSsml":"I wish I was a magician, so I could make [TARGET] disappear."
+  },
+  {
+    "quipGroup":"PEOPLE_OF_WALMART",
+    "text":"You dress like that? Are you trying to end up on the \"People of Walmart\" website?",
+    "ssml":"You dress like that? Are you trying to end up on the People of Walmart website?",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"MASCOT",
+    "text":"I've seen the way you eat. To you, the pig is more than just an animal, it's a personal mascot.",
+    "ssml":"I've seen the way you eat. <break /> To you, the pig is more than just an animal, it's a personal mascot.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"MASCOT",
+    "text":"I've heard the excuses you've given people. To you, the weasel is more than just an animal, it's a personal mascot.",
+    "ssml":"I've heard the excuses you've given people. <break /> To you, the weasel is more than just an animal, it's a personal mascot.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"HEX_NUMBER",
+    "text":"If I had to assign an number to represent you, it would be 2989. Because in hexadecimal that's 0xBAD.",
```

Comment:
It is more technical specific than the others. But how would we ever end up having a flag that unlocks it? Something that asks people if they're technical?